lib/marklogic.js

```diff
@@ -793,6 +793,9 @@ function initClient(client, inputParams) {
       mlutil.copyProperties(inputParams, agentOptions, [
         'keepAliveMsecs', 'maxCachedSessions', 'maxFreeSockets', 'maxSockets', 'maxTotalSockets', 'scheduling', 'timeout'
       ]);
+      agentOptions.secureProtocol = 'TLS_method';
+      agentOptions.secureOptions = require('constants').SSL_OP_NO_TLSv1 | require('constants').SSL_OP_NO_TLSv1_1;
```

Comment:
Is there a reason you're disabling these versions of TLS in the client? I don't think that was the intention of the story.

Reply:
So the default will be 1.3 and the fall back will be 1.2. Disabling version 1 and version 1.1 will make sure the fall back version will be 1.2.

Reply:
These changes are no longer there and as Rob suggested, min and max versions were included.